src/control/actor/host/notify.go

```diff
@@ -187,30 +180,27 @@ func fmtDirection(d network.Direction) string {
 		return "unknown"
 	}
 }
+*/
 
 // Convert from rumor PeerAllData to our Peer. Note that
 // some external data is fetched and some fields are parsed
-func fetchPeerExtraInfo(peerData *track.PeerAllData) (metrics.Peer, error) {
+func fetchPeerExtraInfo(peerData *track.PeerAllData, addr string) (metrics.Peer, error) {
```

Comment:
seems a bit random to have the address here. but regardless of that, its format is not consistent with the one we had before.

i mean:
![image](https://user-images.githubusercontent.com/8811422/134398669-4186b4e6-aebf-4e78-bd0f-12c3a0cdf168.png)

Reply:
so we need to solve:
* use the same address format. either just the ip port. or ipp+port+id
* if you want to use `conn.RemoteMultiaddr().String()`, im good but don't input it to `fetchPeerExtraInfo`

you can just do something like this, and you avoid unnecessarily passing the addrs

```go
peer, err := fetchPeerExtraInfo(peerData)
if err != nil {
	log.Error("Could not fetch peer data for: " + conn.RemotePeer().String())
}
peer.Addrs = conn.RemoteMultiaddr().String()
```